Documentation/Developer/InterlinkInventories.rst

```diff
@@ -33,6 +33,135 @@ This would output:
 *   :ref:`TYPO3 Explained 12 LTS <t3coreapi/v12:start>`
 *   :ref:`TYPO3 Explained 12.4 <t3coreapi/12.4:start>`
 
+..  _available-default-inventories:
+
+Available default inventories
+-----------------------------
+
+These inventories can be used in any rendered documentation:
```

Comment:
Maybe: "... can be used by default in any rendered ..."

This makes it clearer that there is no manual configuration needed.

Reply:
Ack, good idea!